src/actions/index.js

```diff
@@ -24,3 +24,12 @@ export function fetchWeatherByZip(zip) {
 // export const fetchForecast = options => dispatch => {
 //   // return fetch API call
 // }
+
+// hardcode some cities before worrying about API data
+export function fetchWeatherByZip() {
+  console.log("in fetchWeatherByZip function");
```

Comment:
`console.log` is great for debugging but should be removed before merging into master.